### ISSUE TITLE
Special characters on button names throughout site

### ISSUE THREAD

**Issue by dwvanstone** [buttonmen-dev/buttonmen#2556]
The buttons Zoe and Paul Erdos should have special HTML characters in their names. These special characters aren't appearing consistently throughout the site (beyond just the descriptions).

See #2531

**Comment by jl8e** [comment]
That would make it hard to reference them in the message boards, the dynamic search in create game, and anywhere else somebody needs to type a button name for the site to interpret. 

It also means we need to make sure that html entities are interpreted in every context that their names are displayed

**Comment by blackshadowshade** [comment]
Indeed, I had a brief look at what would be necessary. I thought it would be as easy as adding either a couple of rules to the PHP or an extra field in the DB, but as Julian says, there's all the corner cases that don't get directly generated by us, like in the message boards.

I'm tempted to say that if we change it anywhere, we restrict the change to the button page only, so that when viewing buttons.html?button=Zoe, we would change the displayed name in bold, but nothing else on the site.
